bin/krun-sol

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 contract="$1"
 txn="$2"
-shift; shift
-krun -d "$(dirname "$0")/../solidity-kompiled" "$contract" -cTXN="$txn" -pTXN="$(dirname "$0")/kparse-txn" --no-expand-macros "$@"
+isuniswap="$3"
+shift; shift; shift
```

Comment:
let's make krun-sol optionally take a third argument rather than making it required, and let's make the tests in test/regression automatically not pass that optional value, rather than adding a bunch of additional files to the test suite and breaking the interface of `krun-sol`.

Reply:
Addressed. krun-sol has now an optional 3rd argument, that defaults to false. The tests in regression do not pass this argument.